securedrop_client/gui/widgets.py

```diff
@@ -1474,53 +1477,16 @@ def _update_text(self, message_id: str, text: str) -> None:
             self.message.setText(text)
 
 
-class ConversationWidget(QWidget):
```

Comment:
was able to achieve this by passing alignment when adding widgets to the layout (removes code [yay] and simplifies what MessageWidgets and ReplyWidgets are: they are SpeechBubbles, not ConversationWidgets that have SpeechBubbles)